bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -103,9 +103,9 @@ thing-type.shelly.shellypluswdus.description = Shelly Wall Dimmer US Device
 thing-type.shelly.shellyplus10v.description = Shelly Plus Dimmer 10V
 
 # Plus Mini Devices
-thing-type.shelly.shellyplusmini1.description = Shelly Plus Mini 1 - Single Relay Switch
-thing-type.shelly.shellyplusminipm.description =  Shelly Plus Mini PM - Power Meter
-thing-type.shelly.shellyplusmini1pm.description =  Shelly Plus Mini 1PM - Single Relay Switch with Power Meter
+thing-type.shelly.shellymini1.description = Shelly Plus Mini 1 - Single Relay Switch
+thing-type.shelly.shellyminipm.description =  Shelly Plus Mini PM - Power Meter
+thing-type.shelly.shellymini1pm.description =  Shelly Plus Mini 1PM - Single Relay Switch with Power Meter
```

Comment:
Just to be sure, the previous entries were wrong ?
But you did not change thing types ?

Reply:
yes, was thing shelly1mini, which referenced thing-type.shelly.shellyplusmini1.description instead of thing-type.shelly.shellymini1.description etc.